src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/sanity_checks.py

```diff
@@ -64,6 +65,16 @@ def run_sanity_checks(settings):
             "Please select a different ViewLayer.".format(settings.view_layer_selection)
         )
 
+    # Ensure the specified output directory exists
+    output_dir_as_path = Path(settings.output_path)
+    if not (output_dir_as_path.exists() and output_dir_as_path.is_dir()):
+        raise RuntimeError(
+            "The output directory ({}) does not exist.\n"
+            "Please choose an output directory in the submitter dialog, the scene settings, or your preferences.".format(
+                settings.output_path
+            )
```

Comment:
This is after we've grabbed the output location from the scene/preferences, and after a user has specified a folder path? 

This seems like something we can (should?) solve for the user by creating the folder, assuming we have a path they've specified. Thoughts?

Reply:
Sure thing, that makes sense. I'd add a confirmation dialog to this effect after the user hits submit :)